jsonthebuilder/src/components/ResumeField.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import myField from "./ResumeField.scss";
+
+export default function ResumeField(props) {
+
+  
+
+  return (
+    <li className={myField}>
+      {props.name}{<br></br>}
+      {props.label}{<br></br>}
+      {props.email}{<br></br>}
+      {props.website}{<br></br>}
+      {props.location.city}, {props.location.region}{<br></br>}
+      {props.work}
+    </li>
```

Comment:
Not sure pros and cons of either way, but maybe try an <li> element for each line instead of the line break tags?